iodata.rb

```diff
@@ -0,0 +1,34 @@
+$LOAD_PATH << '.'
+require 'file_link'
+require 'csv'
+class IOData
```

Comment:
Nice break down. Please check rubocop for adding new lines before class or method definition for better readability